src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -1331,7 +1331,7 @@ MavlinkMissionManager::parse_mavlink_mission_item(const mavlink_mission_item_t *
 	    mavlink_mission_item->frame == MAV_FRAME_GLOBAL_RELATIVE_ALT ||
 	    (_int_mode && (mavlink_mission_item->frame == MAV_FRAME_GLOBAL_INT ||
 			   mavlink_mission_item->frame == MAV_FRAME_GLOBAL_RELATIVE_ALT_INT))) {
-		// This is a mission item with a global coordinate
+		/** Mission item with a global coordinate **/
```

Comment:
Can you move unrelated changes into a separate commit? And please don't add another type of comments.
```suggestion
		// Mission item with a global coordinate
```

Reply:
I wanted to emphasize the section by having /**/ comment, since it is a very long if statement.
What is the downside of having this type of comment?

Reply:
It's doxygen-style, so used for a different purpose. Plus imo such somewhat random formattings don't help with consistency. In most places we're moving to single-line comments (`//`).